dev/io.openliberty.microprofile.reactive.streams.operators30.internal/src/io/openliberty/microprofile/reactive/streams/operators30/spi/impl/StreamRunner.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2019 IBM Corporation and others.
```

Comment:
Date, if this file had meaningful changes.

Reply:
No changes other than package